.github/workflows/legacy-os.yml

```diff
@@ -16,3 +16,13 @@ jobs:
         run: |
           { { ! sudo scripts/lib/upgrade-zulip-stage-2 2>&1 >&3; } | tee upgrade.err; } 3>&1 >&2
           grep -Fq 'upgrade-zulip-stage-2: Unsupported platform: ubuntu 16.04' upgrade.err
+
+      - name: Report Status
+        if: always()
+        uses: amanagr/notify-zulip-action@master
```

Comment:
That action looks unnecessarily complicated, with its own separate docker image and all. Why not just use a simple `curl` command or Python script?

If we’re going to refer to an external action, we should move it to the `zulip` organization and specify a tag, not `master`.